common/static/common/templates/components/paging-header.underscore

```diff
@@ -4,8 +4,8 @@
     </div>
     <nav class="pagination pagination-compact top">
         <ol>
-            <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label"><%= gettext("Previous") %></span></a></li>
-            <li class="nav-item next"><a class="nav-link next-page-link" href="#"><span class="nav-label"><%= gettext("Next") %></span> <i class="icon fa fa-angle-right"></i></a></li>
+            <li class="nav-item previous"><a class="nav-link previous-page-link" href="#"><i class="icon fa fa-angle-left"></i> <span class="nav-label"><%- gettext("Previous") %></span></a></li>
```

Comment:
These should probably be ```<buttons>``` and not links.  A good rule of thumb is that if it does something, its a button, if it goes somewhere its a link.  Whenver I see href="#" it usually means it should be a button (BTW, # is an invalid value for the href attribute).